.env.example

```diff
@@ -1,4 +1,4 @@
-MONGO_URL=
+MONGO_URL=mongodb+srv://bandvov:z8tVhud7CW7v3rRK@cluster0-f8dbj.azure.mongodb.net/horondi-test?retryWrites=true&w=majoritys
```

Comment:
You should not leave it here. Make an example of it

Reply:
fixed